src/llmcompressor/flows.py

```diff
@@ -0,0 +1,60 @@
+from typing import Dict, Optional, Union
+
+
+def quantize_w8a8(
+    model_id: str,
+    dataset: str,
+    output_dir: str,
+    splits: Dict[str, str],
+    max_seq_length: int,
+    pad_to_max_length: bool,
+    num_calibration_samples: int,
+    smoothquant_kwargs: Optional[Dict[str, Union[int, float, str]]] = None,
+    gptq_kwargs: Optional[Dict[str, Union[int, float, str]]] = None,
```

Comment:
I don't love this UI of having to specify the args as individual dictionaries. If the user is manually setting parameters I think its easier for them to just write out 

```
    recipe = [
        SmoothQuantModifier(....),
        GPTQModifier(....),
    ]
```
rather than calling this function. I would simplify this to only accept smoothing strength and a preset string (ie "FP8", "W8A8" etc)

Reply:
What about model-specific arguments like `mappings`?

Reply:
At that point I just don't see the benefit of having this be a separate function, its not really saving any lines of code. I'll be all for it if we had a preset dictionary and only passed in a preset name though